electrode-ota-server-dao-mariadb/src/dao/PackageDAO.ts

```diff
@@ -248,6 +248,10 @@ export default class PackageDAO extends BaseDAO {
 
     public static async savePackageContent(connection: IConnection, packageHash: string,
         content: Buffer): Promise<void> {
+        const contentResult = await PackageDAO.getPackageContentFromDB(connection, packageHash);
+        if (contentResult && contentResult.length > 0) {
+            return contentResult[0];
```

Comment:
this and 255 return different values.

Reply:
reverted this change and made this in the place we call it.